detox/local-cli/test.js

```diff
@@ -235,8 +239,19 @@ module.exports.handler = async function test(program) {
       ...getPassthroughArguments(),
     ]).join(' ');
 
-    log.info(command);
-    cp.execSync(command, { stdio: 'inherit' });
+
```

Comment:
You could extract the repetitive code to `exec(command, detoxEnvironmentVariables)`.

```
log.info(printEnvironmentVariables(detoxEnvironmentVariables) + command);
cp.execSync(command, {
      stdio: 'inherit',
         env: {
           ...process.env,
           ...detoxEnvironmentVariables
         }
       });
```